example/main.dart

```diff
@@ -1,7 +1,7 @@
 import 'dart:convert' as convert;
 import 'package:http/http.dart' as http;
 
-void main(List<String> arguments) async {
+Future<void> main(List<String> arguments) async {
```

Comment:
Let's drop this lint instead. There are too many valid reasons to violate it.

Reply:
Really? I think these are fine.

Reply:
I forwarded you an email with some discussion about this lint and why it will never be included in `package:pedantic`.

This is not one I want to enforce in any Dart team owned packages.